pkg/ebpf/c/kconfig.h

```diff
@@ -1,6 +1,8 @@
 #ifndef __KCONFIG_H
 #define __KCONFIG_H
 
+#include "bpf_metadata.h"
```

Comment:
Not sure if this is the best place for adding this...
Maybe @brycekahle could weigh in?

Maybe we can explicitly add this everywhere it is needed?

Reply:
I saw that `kconfig.h` and `ktypes.h` were the places where we would cover all programs with the least amount of includes. We could add the include to each program but it seemed to me to be a lot of noise, and also easy to forget or remove.

Reply:
`kconfig.h` would not be included in CO-RE programs, for example.